v2/tools/generator/internal/codegen/pipeline/report_resource_versions.go

```diff
@@ -333,6 +333,8 @@ func (report *ResourceVersionsReport) writeGroupSections(
 	if err != nil {
 		return errors.Wrapf(err, "writing prerelease resources for group %s", group)
 	}
+	// Prerelease may have had no usage, but consume its fragment anyway
+	report.findFragment("prerelease")
```

Comment:
Add a blank line before this to separate it from the error handling immediately above. #nit